river/linear_model/glm.py

```diff
@@ -20,6 +20,7 @@ def __init__(
         optimizer,
         loss,
         l2,
+        l1,
```

Comment:
You'll have to document this new parameter

Reply:
Ofc